.orchestra/config/components/toolchain/msvc.yml

```diff
@@ -330,12 +330,27 @@ toolchain/(@= name @)/vc:
         done
         sleep 0.2
 
-        # Run cl.exe, suppress its output in case of success
-        wine "$SCRIPT_DIR"/(@= bin @)/cl.ex*e "$@"
+        wine "$SCRIPT_DIR"/(@= bin @)/!!NAME!!.ex*e "$@"
 
         EOF
 
+        cp cl.exe dumpbin.exe
+        cp cl.exe ml.exe
+
+        perl -pi -e 's/!!NAME!!/cl/g' "cl.exe" > /dev/null
+        perl -pi -e 's/!!NAME!!/dumpbin/g' "dumpbin.exe" > /dev/null
```

Comment:
* Use `sed`
* As a placeholder use `%NAME%`. Alternatively, consider parsing `$0`, no need to change the script at all in the case.

Reply:
Done (`$0` parsing)